xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/general.less

```diff
@@ -175,7 +175,8 @@ body.content {
       & .xtree,
       & .xwikitabbar,
       & .buttonwrapper,
-      & #user-menu-col {
+      & #user-menu-col,
+      & .xwiki-livedata .cell .displayer-link {
```

Comment:
So the breadcrumb in the location column in Live Data would remain underlined? Seems inconsistent to me.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3085/commits/4b4d7f2b6043f06152384b12f8fad2cef15379d0 :+1: 
Here's what the Page index looks like with the changes proposed in this PR:
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/5746eb84-a774-4bb3-86a8-413479c62552)

Note: I just noticed the underline on the livedata pagination, I don't know what we want to do with that one.